### DISCUSSION TITLE
implementation of a pressure sensor

### DISCUSSION THREAD

**Discussion by luckbert** [mak-gitdev/HA_enoceanmqtt#87] [Request] [answered]
Hello all, 
I am currently trying to switch my home automation from FHEM to Homeassistant. 

It fails just at the implementation of a pressure sensor in the drain pipe. It reports if there is a backwater or not. 
It is my only enocean device.
I am overwhelmed with setting up a new device and need help. In the best case ready files that I just insert. 
I could already find the ID and the EEP herrausfinden. 
ID: 05:18:9A:52
EEP: A5-30-04

I hope that you can help me. 
Greetings Juergen

**Comment by mak-gitdev** [comment]
Hi @luckbert,

What is the reference of your device ?

**Comment by luckbert** [comment]
Sorry, I don't understand the question. 
What do you mean by reference?

**Reply by mak-gitdev** [reply]
@luckbert,

I mean which device is it ? Manufacturer ? Name ? ...

**Comment by luckbert** [comment]
The device is manufactured by "Kessel". And is called "Staufix Control".

The only data I could find is, see attachment.

Grüße Jürgen


![Screenshot_20231021-200015~2](https://github.com/mak-gitdev/HA_enoceanmqtt/assets/30090064/9005e5fd-a078-47a0-89cd-029daae9fbd7)

Screenshot from FHEM

![Screenshot_20231021-201226~2](https://github.com/mak-gitdev/HA_enoceanmqtt/assets/30090064/bb5940bf-011c-4da2-8861-a32c199d0ef4)

**Comment by mak-gitdev** [comment, marked as answer]
@luckbert 

A5-30-03 and A5-30-04 will be available in the next release.

[Follow](https://github.com/users/mak-gitdev/projects/1/views/1?pane=issue&itemId=42272125)

**Comment by luckbert** [comment]
Mega cool,
Thanks a lot!

**Comment by mak-gitdev** [comment]
[Done](https://github.com/users/mak-gitdev/projects/1/views/1?pane=issue&itemId=42272125) in [0.1.27](https://github.com/mak-gitdev/HA_enoceanmqtt/commit/0d0a4cdb3d74b191feb0cc9169e11f10243d1ec9)

**Comment by luckbert** [comment]
Hello, 
I tried to add the sensor today. Unfortunately this did not work. The sensor is not paired with any device. (According to the flashing sequence of the sensor). 
HA_enoceanmqtt is in learn mode.
In the log it says anyway 
"2023-10-23 09:47:33,998 INFO: unknown sensor: 05:18:9A:52

What am I doing wrong?

**Reply by mak-gitdev** [reply]
Hi @luckbert,

Did you add the sensor in your device file ?
Please have a look at this [wiki page](https://github.com/mak-gitdev/HA_enoceanmqtt/wiki/Normal-Usage) for more details.

Please feel free to make comments if something in the procedure is not clear.

Thanks.

**Comment by luckbert** [comment]
Good evening, 
The problem was on me. I have a repeater to increase the range. 
I had to bypass it for the setup and turn it off and be closer to the sensor. 
Now it seems to work. 

Thanks again for the support

Greetings Juergen
